web/config/urls.py

```diff
@@ -14,12 +14,24 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
+from django.conf import settings
+from django.conf.urls.static import static
 from django.urls import path, include
+
 from home.views import HomeView
 
+
 urlpatterns = [ # pylint: disable=C0103
     path('', HomeView.as_view(), name='home'),
     path('admin/', admin.site.urls),
     path('accounts/', include('allauth.urls')),
     path('polls/', include('polls.urls')),
+    path('poker_games/', include('poker_games.urls')),
 ]
+
+
+if settings.DEBUG:
+    urlpatterns = (
+        urlpatterns
+        + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    )
```

Comment:
This must move from here to a separate `urls` file that only called on running the development sever.